notebooks/official/vizier/get_started_vertex_vizier.ipynb

```diff
@@ -861,32 +812,32 @@
         "\n",
-        "-`package_uris`: This is a list of the locations (URIs) of your Python training packages to install on the provisioned instance. The locations need to be in a Cloud Storage bucket. These can be either individual Python files or a zip (archive) of an entire package. In the later case, the job service will unzip (unarchive) the contents into the docker image.\n",
+        "    - `package_uris`: This is a list of the locations (URIs) of your Python training packages to install on the provisioned instance. The locations need to be in a Cloud Storage bucket. These can be either individual Python files or a zip (archive) of an entire package. In the later case, the job service unzips (unarchives) the contents into the docker image.\n",
         "\n",
-        "-`python_module`: The Python module (script) to invoke for running the custom hyperparameter tuning job. In this example, you will be invoking `trainer.task.py` -- note that it was not neccessary to append the `.py` suffix.\n",
+        "    - `python_module`: The Python module (script) to invoke for running the custom hyperparameter tuning job. In this example, invoke the `trainer.task.py` -- note that it isn't neccessary to append the `.py` suffix.\n",
         "\n",
-        "-`args`: The command line arguments to pass to the corresponding Python module. In this example, you will be setting:\n",
-        "  - `\"--model-dir=\" + MODEL_DIR` : The Cloud Storage location where to store the model artifacts. There are two ways to tell the hyperparameter tuning script where to save the model artifacts:\n",
-        "      - direct: You pass the Cloud Storage location as a command line argument to your training script (set variable `DIRECT = True`), or\n",
-        "      - indirect: The service passes the Cloud Storage location as the environment variable `AIP_MODEL_DIR` to your training script (set variable `DIRECT = False`). In this case, you tell the service the model artifact location in the job specification.\n",
-        "  - `\"--epochs=\" + EPOCHS`: The number of epochs for training.\n",
-        "  - `\"--steps=\" + STEPS`: The number of steps (batches) per epoch.\n",
-        "  - `\"--distribute=\" + TRAIN_STRATEGY\"` : The hyperparameter tuning distribution strategy to use for single or distributed hyperparameter tuning.\n",
-        "     - `\"single\"`: single device.\n",
-        "     - `\"mirror\"`: all GPU devices on a single compute instance.\n",
-        "     - `\"multi\"`: all GPU devices on all compute instances."
+        "    - `args`: The command line arguments to pass to the corresponding Python module. In this example, you're going to configure\n",
+        "    \n",
+        "        - `--model-dir` : The Cloud Storage location where to store the model artifacts. There are two ways to tell the hyperparameter tuning script where to save the model artifacts:\n",
```

Comment:
add "for": 
- "The Cloud Storage location for where to store the model artifacts."